MainRepository/Program.cs

```diff
@@ -1,7 +1,25 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿using System.Reflection;
+
+//Strong password
+using (var stm = Assembly.GetExecutingAssembly ().GetManifestResourceStream ("MainRepository.Data.passwords.txt"))
+using (var sr = new StreamReader (stm)) {
+   while (!sr.EndOfStream) {
+      var result = sr.ReadLine ();
+      Console.WriteLine ($"The given {result} password is " + (isValidPassword (result) ? "Strong" : "Weak"));
+   }
+}
+static bool isValidPassword (string s) {
+   bool containsUpperCase = false;

```

Comment:
If you are concerned only for validness of password, not many bool flags are required. You could have used single return statement with conditions.

Since you are using flags for each checks, Can you print error messages accordingly. (Eg: "Atleast one UpperCase required, Atleast one SpecialChar required, ...)?